.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  presets: ["env", "flow"]
+  presets: ["env", "flow"],
+  plugins: ["transform-runtime"]
```

Comment:
what is this?

Reply:
This is to allow to use await/async in the code.

Reply:
really?
weird.

Reply:
Some things are not available on default babel and need this or `babel-polyfill` or some other plugins. This relates to #26